doc/development_guide.md

```diff
@@ -12,134 +13,336 @@ a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow th
-        export MSSQL_CLI_PASSWORD=<password>
-    ```
-
-    Execute the unit tests.
-    ```
-    python build.py unit_test
-    ```
-    
-2. Run integration tests (code format, unit tests, packaging, integration,) with tox:
-
-    ```
-    python build.py integration_test
-    ```
-
-## <a name="Run_mssql-cli"></a>5. Running mssql-cli
-#### Command line
-
-1.  Invoke mssql-cli using:
-
-    ##### MacOS/Linux (bash):
-    ```
-    mssql-cli --help
-    ```
-
-    ##### Windows:
-    ```
-    <clone_root>\mssql-cli.bat --help
-    ```
-    which is equivalent to the following:
-    ```
-    python -m mssql-cli --help
-    ```
+![image](https://user-images.githubusercontent.com/19577035/63064795-f928bd00-beb7-11e9-8d53-64840a7a8dcb.png)
+
+In order to activate the python extension, select one of the `*.py` file in the explorer on the left.
+You will see the extension becomes active and showes current version of python hooked up to the extension by default.
+
+![image](https://user-images.githubusercontent.com/19577035/63064822-1eb5c680-beb8-11e9-8f22-0d3ddad25a95.png)
+
+### 2. Configure settings.json
+If `<workspaceRoot>/.vscode` does not exists, change one of the user settings to create it. One of the easiest way is simply changing the current version of python to any version.
+
+![image](https://user-images.githubusercontent.com/19577035/63064397-b9f96c80-beb5-11e9-8514-b0e58e83f3a5.png)
+
+And you will see `<workspaceRoot>/.vscode` and `<workspaceRoot>/.vscode/settings.json` file are created.
+
+![image](https://user-images.githubusercontent.com/19577035/63064590-bca89180-beb6-11e9-8800-1bcde427cef0.png)
+
+Update the `<workspaceRoot>/.vscode/settings.json` as shown below:
+##### Windows
+```
+{
+    "python.pythonPath": "${workspaceRoot}/env/Scripts/python.exe",
+    "python.linting.pylintEnabled": true,
+    "python.envFile": "${workspaceRoot}/environments.env",
+    "python.testing.nosetestsEnabled": true,
+    "python.testing.nosetestArgs": [
+        "tests",
+        "mssqlcli/jsonrpc/tests",
+        "mssqlcli/jsonrpc/contracts/tests"
+    ],
+    "python.testing.pytestEnabled": false,
+    "python.testing.unittestEnabled": false
+}
+```
+##### MacOS / Linux
+```
+{
+    "python.pythonPath": "${workspaceRoot}/env/bin/python",
+    "python.linting.pylintEnabled": true,
+    "python.envFile": "${workspaceRoot}/environments.env",
+    "python.testing.nosetestsEnabled": true,
+    "python.testing.nosetestArgs": [
+        "tests",
+        "mssqlcli/jsonrpc/tests",
+        "mssqlcli/jsonrpc/contracts/tests"
+    ],
+    "python.testing.pytestEnabled": false,
+    "python.testing.unittestEnabled": false
+}
+```
+
+Press `Yes` if any notification comes up for missing dependencies. Please make sure to complete [Build mssql-cli](#Build_mssql_cli) step before moving forward.
+
+![image](https://user-images.githubusercontent.com/19577035/63066150-9b976f00-bebd-11e9-8e31-a786e2eb61f5.png)
+
+
+
+### 3. Create environments.env file

```

Comment:
Now that this is checked in, these instruction should be updated to edit the file.

Reply:
fixed.